python-package/xgboost/core.py

```diff
@@ -254,6 +254,8 @@ def __init__(self, data, label=None, missing=None,
             Set names for features.
         feature_types : list, optional
             Set types for features.
+        nthread : integer, optional
+            Number of threads to use for loading data from numpy array.
```

Comment:
I believe -1 means use all threads. Maybe just add a note for this?

Reply:
Sweet, added that, thanks.